src/site/content/en/handbook/content-checklist/index.md

```diff
@@ -10,9 +10,9 @@ description: |
 This is a checklist for web.dev content reviews. It includes critical high-level issues and lower-level issues that commonly appear in drafts.
```

Comment:
I suggest rephrasing this to be actionable. "Use this checklist to make content reviews go faster" or something. Follow the style guide rule about value statements.

Reply:
So, I didn't make this imperative because it's really only for a subset of handbook readers--the content reviewers. Maybe it should say that?

Reply:
Rephrased in Google Doc